prow/plugins/lgtm/lgtm.go

```diff
@@ -471,6 +472,29 @@ func handlePullRequest(log *logrus.Entry, gc githubClient, config *plugins.Confi
 	return gc.CreateComment(org, repo, number, removeLGTMLabelNoti)
 }
 
+func removeLGTMAndRequestReview(gc githubClient, org, repo string, number int, logins []string, storeTreeHash bool) error {
+	if err := gc.RemoveLabel(org, repo, number, LGTMLabel); err != nil {
+		return fmt.Errorf("failed removing lgtm label: %w", err)
+	}
+
+	// Re-request review because LGTM has been removed only if storeTreeHash enabled.
+	// TODO(mpherman): Surface User errors to PR
+	if storeTreeHash {
```

Comment:
@BenTheElder Revisiting this PR. Added an extra check to reduce notification spam on PRs that do not have storeTreeHash enabled. Thoughts?

Reply:
seems like a good plan, hopefully we can also get this enabled widely (perhaps even by default at some point?)